### ISSUE TITLE
Regarding parallel coordinates

### ISSUE THREAD

**Issue by pytholic** [allegroai/clearml#907]
Hi. 
I am testing hyperparameter optimization with different parameters and models.

![image](https://user-images.githubusercontent.com/60126503/217999604-4ec0a9ec-41e9-4c05-bdeb-d0914f808880.png)

Everything looks good except the part where the `Args/net` column is. I passed two different models like:
```python
optimizer = HyperParameterOptimizer(
    base_task_id="0467f868e9154f3ab72727c141614d52",
    hyper_parameters=[
        UniformParameterRange("Args/lr", min_value=0.01, max_value=0.3, step_size=0.05),
        DiscreteParameterRange("Args/net", values=["Net1", "Net2"]),
        DiscreteParameterRange("Args/epochs", values=[5, 10]),
    ],
```
So all the tasks should be matched to either one of these nets (as shown in Args of each experiment). However, this mapping does not seem correct.

**Comment by ainoam** [comment]
Thanks for reporting @pytholic, I think there's an issue with the visualization but the actual experiments indeed only used on of the two values specified. Can you confirm?

**Comment by pytholic** [comment]
> Thanks for reporting @pytholic, I think there's an issue with the visualization but the actual experiments indeed only used on of the two values specified. Can you confirm?

Yes @ainoam , that is exactly the issue. It works fine, but I was just wondering why the visualization is little off specifically at that point.

**Comment by ainoam** [comment]
Looks like there might be an issue with how the plot gets created - We'll update as a fix becomes available.

**Comment by pytholic** [comment]
@ainoam All right. Thank you for the response!

**Comment by pollfly** [comment]
Hey @pytholic! Just letting you know that this issue has been resolved in the recently released [v1.10.0](https://github.com/allegroai/clearml/releases/tag/v1.10.0). Let us know if there are any issues :)

**Comment by pytholic** [comment]
@pollfly thank you for the update!
